packages/libs/wallet-sdk/src/sdk/host.ts

```diff
@@ -9,32 +14,64 @@ export type ChainwebHostGenerator = (options: {
 /**
  * @public
  */
-export type GraphqlHostGenerator = (options: { networkId: string }) => string;
+export type GraphqlHostGenerator = (options: {
+  networkId: string;
+  graphType?: GraphType;
+}) => string;
 
+// Kadena GraphQL Host Map
+const kadenaGraphqlHostMap: Record<string, string> = {
+  mainnet01: 'https://graph.kadena.network/graphql',
+  testnet04: 'https://graph.testnet.kadena.network/graphql',
+};
+
+// Hackachain GraphQL Host Map
+const hackachainGraphqlHostMap: Record<string, string> = {
+  mainnet01: 'https://www.kadindexer.io/graphql',
+  testnet04: 'https://www.kadindexer.io/graphql',
+};
```

Comment:
Is this determined by the consumer or in the library itself?
Can the consumer override it?